test/integration/scripting_spec.js

```diff
@@ -534,10 +534,15 @@ describe("Interaction", () => {
                 page,
                 getSelector(refOpen),
                 async () => {
+                  const sel = getSelector(refOpen);
+                  await page.waitForSelector(sel, {
+                    timeout: 0,
```

Comment:
Is this safe in the sense that if this selector would not trigger we would basically be waiting here forever? Or is there a global timeout on the Jasmine tests or so?